src/dvc_objects/db.py

```diff
@@ -50,8 +51,16 @@ def _init(self):
         if self._initialized:
             return
 
-        for num in range(0, 256):
-            self.makedirs(self.fs.path.join(self.path, f"{num:02x}"))
+        dnames = {f"{num:02x}" for num in range(0, 256)}
+        existing = set()
+        with suppress(FileNotFoundError):
```

Comment:
Should probably also catch `NotImplementedError`.

Reply:
And other `OSError`s too?

Reply:
Ah, right, for http. Can see a test failure now too.